student-bios/MadhuMNG.md

```diff
@@ -0,0 +1,33 @@
+# Hello everyone
+
+I am Madhu.I am basically from India. Since 5 years I am staying in Belgium.
+
+## Linguistic capabilities
+
+1. Kannada
+2. English
+3. French and Dutch
+
+### Education
+
+- I have done Master in Business Adninistration from Bangalore University
+- I am currenting learning French-B1 level
+- I am a student in HYF
+
+## Work Experience
+
+| Organisation | Role                                             |
+| ------------ | ------------------------------------------------ |
+| Ernst&Young  | Advanced tax analyst                             |
+| TCS          | Project managing officer , Subject matter expert |
+
+### Hobbies
+
+:joy: I am passion about Artistic activities such as painting, pencil sketch,
+crochet, Resin jewellery making
+
+[My insta page] https://www.instagram.com/padma_vibhooshita/
```

Comment:
Hey @MadhuMNG , you can take a look at this line again, what do we need for links to work? You can also check the preview option on VScode to see if your code works the way you wanted it to work.